apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -18,10 +27,78 @@ FluentUI supports browsers that are at or above the indicated versions below:
 | ------------- | ------------------ | ------- |
 | >=14.5        | >=84               | >=16    |
 
-FluentUI will ensure that no browser, EcmaScript(ES), or CSS features that are incompatible with the support matrix laid out above are introduced.
-FluentUI will not be providing polyfills for features that we expect our browser support matrix to already support. This means that consumers who require the
-use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`),
-are expected to implement their own polyfills or use a third-party one.
+FluentUI will ensure that no browser, ECMAScript, or CSS features that are incompatible with the support matrix laid out above are introduced.
+For consumers who require the use/support of older browsers, please see the partial support matrix and polyfill sections below for more information
+on what will need to be done.
+
+## Partial Browser Support Matrix:
+
+FluentUI introduces the concept of a partial browser support matrix as a means to find that balance between supporting modern browsers and older browsers.
+The partial browser matrix below will have limitations that will require consumers to transpile down and/or use polyfills to cover the support gaps. The feature gaps
+are listed below with subsequent recommendation on how to handle each case.
+
+**Desktop Browsers:**
+
+| Edge | Firefox | Chrome | Safari | Opera | Internet Explorer |
+| ---- | ------- | ------ | ------ | ----- | ----------------- |
+| >=79 | >=69    | >=79   | >=13.1 | >=64  | Not Supported     |
+
+**Mobile Browsers:**
+
+| Safari on iOS | Chrome for Android | Samsung |
+| ------------- | ------------------ | ------- |
+| >=13.4        | >=79               | >=14    |
+
+List of features that are currently used in FluentUI that aren't supported out of the box by the partial support matrix above:
+
+**Unsupported CSS Features:**
+
+- Support for the CSS features below are lost within the partial support matrix. The lost of these features will not crash consumer applications,
+  it will simply degrade the user interface.
+
+1.  Flex gap: Unsupported by Chrome, Edge, Firefox, Opera, Safari, Safari on IOS, Chrome on Android
+2.  CSS min, max, clamp: Unsupported by Firefox and Opera
+3.  CSS revert value: Unsupported by Firefox and Opera
+
+**Unsupported ECMAScript Features:**
+
+- ES2020 features listed below are already being used within the FluentUI library. Consumers who follow the partial support matrix will need to transpile down
+  the FluentUI library to make it work since the use of these ES2020 features will result in the application crashing on some of the listed browsers.
```

Comment:
```suggestion
  the Fluent UI library to make it work since the use of these ES2020 features will result in the application crashing on some of the listed browsers.
```